### PR TITLE
Xmldoc fixes

### PR DESCRIPTION
**Pull request by sean-gilliam**
Fixes for missing tags, extraneous tags, and invalid characters.

### CODE REVIEW

File: src/Shared/FrameworkLocationHelper.cs [hunk 1]
```diff
@@ -559,7 +559,7 @@ internal static string GetPathToDotNetFramework(Version version, DotNetFramework
         /// Check the registry key and value to see if the .net Framework is installed on the machine.
         /// </summary>
         /// <param name="registryEntryToCheckInstall">Registry path to look for the value</param>
-        /// <param name="registryValueToCheckInstall">Key to retrieve the value from</param>
+        /// <param name="registryValueToCheckInstall">Key to retrieve the value from</param>
```
Comment (sarajoiner) [inline src/Shared/FrameworkLocationHelper.cs]: I'm assuming this (and the other handful of visibly unchanged lines) are EOL fixes as mentioned in #22 ?

**Comment by sarajoiner** [comment]
LGTM :+1:

File: src/Shared/FrameworkLocationHelper.cs (hunk 1 above)
Comment (sean-gilliam) [inline src/Shared/FrameworkLocationHelper.cs]: Yea it looks like it

**Comment by dannyvv** [comment]
@sean-gilliam Hi Sean, It seems you hit a merge conflict with someone else. Would you mind resolving the merge conflicts?
Otherwse the change looks good for me too.

**Comment by sean-gilliam** [comment]
okay @dannyvv it should be gtg now.

**Comment by dannyvv** [comment]
Thanks!

**Pull request merged**